docs/pages/installation.mdx

```diff
@@ -108,6 +108,9 @@ describes the available Docker images.
 These images are hosted on [Amazon ECR
 Public](https://gallery.ecr.aws/gravitational).
 
+Starting from Teleport `15.0` only distroless images are provided to increase
+default security.
```

Comment:
I think this is more appropriate for release notes and can be removed here. v15 readers will just see the relevant information.

Reply:
I think we must mention what's the specificity of those images and how to exec commands in them.

Most people don't know what to do when interacting with a shell-less image.

Maybe something like:

---

Since version 15, Teleport images are based on [google's distroless](https://github.com/GoogleContainerTools/distroless). Those images don't contain any shell. Here are some examples on how to invoke commands in shell-less images:

```code
# in docker
$ docker run -i my-container tctl status

# in Kubernetes
$ kubectl exec -i my-pod -- tctl status

# sending local files via stdin
$ kubectl exec -i my-pod -- tctl create -f < my-local-file.yaml

# retrieving output via stdout and tar
$ kubectl exec -i my-pod -- tctl auth sign --user admin --format tls --ttl 10m --tar -o admin| tar xv -C local
$ ls -l local
total 24
-rw-------  1 trent  staff  1318 Jul 24 15:52 admin.cas
-rw-------  1 trent  staff  1895 Jul 24 15:52 admin.crt
-rw-------  1 trent  staff  1679 Jul 24 15:52 admin.key
```

Alternatively, you can use the debug variant of the image, which contains [busybox](https://www.busybox.net/about.html) and a minimal shell invocable via `busybox sh`:

```code
$ docker run -it --entrypoint="" public.ecr.aws/gravitational/teleport-distroless
-debug:(=teleport.latest_oss_docker_image=) busybox sh
```

Reply:
Please see update @hugoShaka

Reply:
If you get a chance to look too @ptgott . thanks